### PR TITLE
[release-4.2] Bug 1788705: Lazy load images in OperatorHub

### PR DESCRIPTION
**Pull request by spadgett**
This addresses some performance problems we're seeing loading images in
OperatorHub. Specifically, the subscribe flow is blocked until all
images from the catalog load. Since we do code splitting, the request
for the JS chunk for the next page is blocked because of the pending
image requests. This has also caused timeouts in our OLM integration
tests in CI.

4.2.z backport of #3293

### CODE REVIEW

**Comment by spadgett** [comment]
I pulled in the additional test fixes from #3650 

/assign @rhamilto

File: frontend/package.json [hunk 1]
```diff
@@ -93,7 +93,7 @@
     "openshift-logos-icon": "1.7.1",
     "patternfly": "^3.59.1",
     "patternfly-react": "2.32.3",
-    "patternfly-react-extensions": "2.18.4",
+    "patternfly-react-extensions": "3.0.2",
```
Comment (rhamilto) [inline frontend/package.json]: This gives me pause since it is a major version bump.  All good?

Comment (spadgett) [reply frontend/package.json]: We need this commit for the `icon` prop: https://github.com/patternfly/patternfly-react/commit/694cd049ee324e07b32333be5406967425002fc8

We could potentially go to any version 2.20+. I picked the one we were using in 4.3.

It looks like the major version bump was for xterm 4.x. It doesn't seem to impact the console xterm version, though.

https://github.com/patternfly/patternfly-react/releases/tag/patternfly-react-extensions%403.0.0

**Pull request merged**